src/drive/mobile/containers/onboarding/SelectServer.jsx

```diff
@@ -9,10 +9,36 @@ import { registerDevice, setUrl } from '../../actions/settings'
 import styles from '../../styles/onboarding'
 
 export class SelectServer extends Component {
+  constructor(props) {
```

Comment:
I forgot about this. Actually, you can just do

```
export class SelectServer extends Component {
state = {fetching: false}
```

and then you don't even need a constructor.

Reply:
oh right ok then :)